data/LAU/.temp.tarql.bak

```diff
@@ -0,0 +1,32 @@
+# Own prefixes
+prefix ebg:               <http://data.eubusinessgraph.eu/ontology#> 
+prefix nuts: <http://nuts.geovocab.org/id/> 
+prefix ramon: <http://rdfdata.eionet.europa.eu/ramon/ontology/> 
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> 
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> 
+prefix xs: <http://www.w3.org/2001/XMLSchema#> 
+prefix spatial: <http://geovocab.org/spatial#> 
+prefix lau:                 <http://data.businessgraph.io/lau/>
+
+construct {
+    ?lau1_ ramon:name     ?latin_name;
+           ramon:name     ?NAME1;
+           ramon:code     ?CODE ;
+           rdfs:label     ?label1;
+} WHERE {
+    BIND("http://data.businessgraph.io/lau/" as ?base)
+    BIND(IF(?LAU1 = "n.a." || ?LAU1 = "" || ?LAU1 = "no", 
+            ?undef,
+            URI(CONCAT(?base, UCASE("uk"), "-", ?LAU1))) as ?lau1_)
+
+    BIND(IF(?NAME2 = "",
+            ?undef,
+            strlang(?NAME2, "en")) as ?latin_name)
+
+    BIND(IF(?NAME_1 = "",
+            ?undef,
+            strlang(?NAME1, "{language}")) as ?native_name)
```

Comment:
it's replaced by the loop.sh and adds the language tag

Reply:
Please include a comment like this. 
(I'd use `$language` for the replacement... a SPARQL convention is to use `$` instead of `?` for externally-bound variables)

Reply:
Done